app/black_box/black_box.rb

```diff
@@ -50,20 +53,13 @@ def calculate_bonus_score(body_markdown)
       size_bonus + code_bonus
     end
 
-    def last_mile_hotness_calc(article, function_caller)
-      if ENV["AWS_SDK_KEY"].present? && ENV["AWS_SDK_KEY"] != "foobarbaz"
-        function_caller.call(
-          "blackbox-production-articleHotness",
-          { article: article, user: article.user }.to_json,
-        ).to_i
-      else
-        # Simple calculation that takes in published at time and scores
-        # Same order of magnitude calculation as existing private function
-        # Gives credit to new articles and articles which score well from users and mods
-        article.published_at.to_i / 10_000 +
-          (article.score * 5000) +
-          (article.comment_score * 5000)
-      end
+    def last_mile_hotness_calc(article)
+      epoch_number = "2010-01-01 00:00:01".to_time.to_i
```

Comment:
as this never changes, maybe we can put it in a constant.

Reply:
Good call.